website_snippet_barcode/static/src/js/website_snippet_barcode.js

```diff
@@ -0,0 +1,64 @@
+/* Copyright 2017 LasLabs Inc.
+ * License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl.html). */
+
+odoo.define('website_snippet_barcode', function (require) {
+    "use strict";
+
+    var animation = require('web_editor.snippets.animation');
+
+    animation.registry.barcode = animation.Class.extend({
+        selector: '.s_barcode',
+
+        defaults: {
+            type: 'QR',
+            value: encodeURIComponent(window.location.href.split('#')[0]),
+            aspectratio: '1',
```

Comment:
Oops I should have noticed this first time around. Should use camelCase for JS:
```
aspectRatio: ...,
humanReadable: ...,
```

Reply:
Oh. Derp. Thanks for catching that.